### PR TITLE
Update "KEP-993: Two Stage Admission Process".

### LINKED ISSUE #2437: Deprecate the `retryDelayMinutes` from the AdmissionCheck API
**What would you like to be added**:
Deprecate the `retryDelayMinutes` from the AdmissionCheck API

**Why is this needed**:
It is not used anywhere in the codebase

**Completion requirements**:

This enhancement requires the following artifacts:

- [ ] Docs update

The artifacts should be linked in subsequent comments.

### PR DESCRIPTION
**Pull request by mbobrovskyi**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Update "KEP-993: Two Stage Admission Process".

#### Which issue(s) this PR fixes:

Fixes #2437

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: keps/993-two-phase-admission/kep.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ title: Two Stage Admission Process
 kep-number: 993
 authors:
   - "@mwielgus"
-status: draft
+status: implemented
```
Comment (mimowo) [inline keps/993-two-phase-admission/kep.yaml:5]: nit: use `implementable`. implemented is basically reserved for GA features. Until then we are free to extend them: https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/0000-kep-process/README.md#kep-workflow

**Review by mimowo — COMMENTED** [review]
LGTM

File: keps/993-two-phase-admission/kep.yaml (hunk 1 above)
Comment (mbobrovskyi) [reply keps/993-two-phase-admission/kep.yaml]: Done

**Comment by mbobrovskyi** [comment]
It's already deprecated on https://github.com/kubernetes-sigs/kueue/pull/2408.

File: keps/993-two-phase-admission/README.md [hunk 2]
```diff
@@ -277,6 +272,16 @@ The tests should cover:
 
 ### Graduation Criteria
 
+#### Deprecation
+
+We deprecate the RetryDelayMinutes, and going to remove it in feature versions of the API.
+
+First iteration (0.8):
```
[the remark this answers is not in the document]
Comment (mbobrovskyi) [reply keps/993-two-phase-admission/README.md:279]: It's already deprecated on https://github.com/kubernetes-sigs/kueue/pull/2408.

[the remark this answers is not in the document]
Comment (tenzen-y) [reply keps/993-two-phase-admission/README.md:279]: Ah, that's right.
Although I reviewed the PR, I forgot it...

**Pull request merged**